dist_run.py

```diff
@@ -304,7 +306,7 @@ def _cleanup():
 
 
 def main(args):
-    model_name = args.model_name
+    model_name = "llama3"  # args.model_name
```

Comment:
I think this is leftover from my branch?  This should be reset back to args.model_name as I was just hardcoding it to avoid dealing with passing params at that time.